pymodbus/repl/client/README.md

```diff
@@ -32,7 +38,9 @@ Commands:
   serial
   tcp
 
+# Or using a Docker run command instead
 
+✗ docker run -it riptideio/pymodbus pymodbus.console --help
```

Comment:
And how about the compose options you wrote about earlier.

Reply:
The compose example does not applies to the REPL, but I can add a particular example to run the REPL using docker